src/components/QueryOutput/QueryOutput.js

```diff
@@ -0,0 +1,1988 @@
+          })
+
+          const shouldFilter = `${formattedElement}`
+            .toLowerCase()
+            .includes(`${headerValue}`.toLowerCase())
+
+          return shouldFilter
+        } catch (error) {
+          console.error(error)
+          this.props.onErrorCallback(error)
+          return false
+        }
+      }
+    } else if (
+      col.type === 'DOLLAR_AMT' ||
+      col.type === 'QUANTITY' ||
+      col.type === 'PERCENT' ||
+      col.type === 'RATIO'
+    ) {
+      return (headerValue, rowValue, rowData, filterParams) => {
+        try {
+          const trimmedValue = headerValue.trim()
+          if (trimmedValue.length >= 2) {
+            const number = Number(
+              trimmedValue.substr(1).replace(/[^0-9.]/g, '')
+            )
+            if (trimmedValue[0] === '>' && trimmedValue[1] === '=') {
+              return rowValue >= number
+            } else if (trimmedValue[0] === '>') {
+              return rowValue > number
+            } else if (trimmedValue[0] === '<' && trimmedValue[1] === '=') {
+              return rowValue <= number
+            } else if (trimmedValue[0] === '<') {
+              return rowValue < number
+            } else if (trimmedValue[0] === '!' && trimmedValue[1] === '=') {
+              return rowValue !== number
+            } else if (trimmedValue[0] === '=') {
+              return rowValue === number
+            }
+          }
+
+          // No logical operators detected, just compare strings
+          const strippedHeader = headerValue.replace(/[^0-9.]/g, '')
+          return rowValue.toString().includes(strippedHeader)
+        } catch (error) {
+          console.error(error)
+          this.props.onErrorCallback(error)
+          return false
+        }
+      }
+    }
+
+    return undefined
+  }
+
+  setSorterFunction = (col) => {
+    if (col.type === 'DATE' || col.type === 'DATE_STRING') {
+      return (a, b) => dateSortFn(b, a)
+    } else if (col.type === 'STRING') {
+      // There is some bug in tabulator where its not sorting
+      // certain columns. This explicitly sets the sorter so
+      // it works every time
+      return 'string'
+    }
+
+    return undefined
+  }
+
+  formatColumnsForTable = (columns) => {
+    // todo: do this inside of chatatable
+    if (!columns) {
+      return null
+    }
+
+    const formattedColumns = columns.map((col, i) => {
+      /**
+       * EDIT:
+       * We no longer want to default to one over the other. Howeever,
+       * I would like to hang onto this code for now incase we do want to
+       * include either/or in some cases in the future
+       */
+      // Regardless of the BE response, we want to default to percent
+      // if (
+      //   (col.type === 'RATIO' || col.type === 'NUMBER') &&
+      //   _get(
+      //     getDataFormatting(this.props.dataFormatting),
+      //     'comparisonDisplay'
+      //   ) === 'PERCENT'
+      // ) {
+      //   col.type = 'PERCENT'
+      // }
```

Comment:
I think I left this. We can probably remove it unless you feel there might be a reason to keep?